pontoon/base/urls.py

```diff
@@ -60,7 +60,7 @@
         name='pontoon.translate'),
 
     # List contributors
-    url(r'^contributors/$',
+    url(r'^contributors/(?P<period>overall|12months|6months|3months|lastmonth)$',
```

Comment:
I think we should not include these in the base URL. The _thing_ that users are accessing is the same (a list of contributors), it's just limited to a certain period of time. 

It'd be better, I think, to use a querystring parameter, and keep this URL as it is. The querystring parameter can default to the `overall` view so that we don't have to issue a redirect.